packages/types/src/index.ts

```diff
@@ -12,16 +12,28 @@ export interface Order {
     makerAssetData: string;
     takerAssetData: string;
     salt: BigNumber;
+    exchangeAddress: string;
```

Comment:
It should we extending `OrderWithoutExchangeAddress`

Reply:
Agreed, but we won't do that because `OrderWithoutExchangeAddress` is a internal implementation detail, we don't want it showing up in the docs.